src/main/scala/org/broadinstitute/dsde/snoop/SnoopConfig.scala

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.dsde.snoop
```

Comment:
I don't mind this class but it should encompass all config and should NEVER be referenced outside of Boot.scala